frontend/src/components/common/SelectField/SelectField.tsx

```diff
@@ -1,31 +1,31 @@
-import React from 'react';
-import './Input.css';
+import './SelectField.css';
 
 interface InputProps {
   name: string;
-  type: 'string' | 'email' | 'password' | 'number';
   placeholder: string;
-  onChange: (e: React.FormEvent<HTMLInputElement>) => void;
+  onChange: (e: React.FormEvent<HTMLSelectElement>) => void;
   value?: string;
   label?: string;
 }
 
-export function Input({ name, type, placeholder, value, onChange, label }: InputProps) {
+export function SelectField({ name, placeholder, value, label, onChange }: InputProps) {
   return (
     <div className="flex-box">
       <label htmlFor={name} className="edit-profile-label">
         {label}
       </label>
-      <input
-        className="input-edit-field"
-        type={type}
+      <select
+        className={label ? 'input-edit-field' : 'input-field'}
         name={name}
         placeholder={placeholder}
-        onChange={onChange}
         autoComplete="off"
         value={value}
         id={name}
-      />
+        onChange={onChange}
+      >
+        <option value="male">male</option>
+        <option value="female">female</option>
```

Comment:
With these options, this component is not reusable. I can only use it for selecting genders.

Update this to be more reusable.